apis/datadoghq/v1alpha1/datadogagent_types.go

```diff
@@ -56,6 +56,8 @@ type DatadogAgentSpec struct {
 
 	// Set a unique cluster name to allow scoping hosts and Cluster Checks Runner easily.
```

Comment:
Let's include the pattern requirements in the description

```suggestion
	// Set a unique cluster name to allow scoping hosts and Cluster Checks Runner easily. It must be dot-separated tokens where tokens start with a lowercase letter followed by lowercase letters, numbers, or hyphens, cannot end with a hyphen nor have a dot adjacent to a hyphen, and be below 80 chars.
```